consensus-types/epbs/indexed_payload_attestation_test.go

```diff
@@ -0,0 +1,60 @@
+package epbs
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	eth "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+)
+
+func TestGetAttestingIndices(t *testing.T) {
+	attestingIndices := []primitives.ValidatorIndex{1, 2, 3}
+	nilAttestingIndices := []primitives.ValidatorIndex{}
+	pa := &IndexedPayloadAttestation{
+		AttestingIndices: attestingIndices,
+	}
+	got := pa.GetAttestingIndices()
+	for i, v := range got {
+		require.Equal(t, attestingIndices[i], v)
+	}
+	pa = &IndexedPayloadAttestation{
+		AttestingIndices: nilAttestingIndices,
+	}
+	got = pa.GetAttestingIndices()
+	require.DeepEqual(t, nilAttestingIndices, got)
+}
+
+func TestGetData(t *testing.T) {
+	data := &eth.PayloadAttestationData{
+		Slot: 1,
+	}
+	nilData := &eth.PayloadAttestationData{}
```

Comment:
Same here. It's not nil. It has a nil slice for BeaconBlockRoot and 0 for Slot and PayloadStatus respectively, which is their default value.